assignments/closure.js

```diff
@@ -16,4 +42,27 @@ const counterFactory = () => {
   // Return an object that has two methods called `increment` and `decrement`.
   // `increment` should increment a counter variable in closure scope and return it.
   // `decrement` should decrement the counter variable and return it.
+  let count = 0;
+  return {
+    increment: function() {
```

Comment:
Both `increment` and `decrement` should return the variable after changing it. You can do this by returning count after incrementing/decrementing, or you can use `return ++count` as a one-liner, which increments `count` before returning it.